templates/etc_systemd_system_postgresql.service.d_custom.conf.j2

```diff
@@ -8,4 +8,8 @@ Group={{ postgresql_service_group }}
 Environment=PGDATA={{ postgresql_conf_directory }}
 
 ExecStartPre=
+    {% if postgresql_version == 10 %}
+ExecStartPre={{ postgresql_bin_directory }}/postgresql-{{ postgresql_version_terse }}-check-db-dir {{ postgresql_data_directory }}
+    {% else %}
```

Comment:
Is the indentation on this correct?  Should it be better written in this format, ans using the `version_check()` builtin:
```
{% if postgresql_version | version_check('10', '>=') %}
  ExecStartPre={{ postgresql_bin_directory }}/postgresql-{{ postgresql_version_terse }}-check-db-dir {{ postgresql_data_directory }}
{% else %}
  ExecStartPre={{ postgresql_bin_directory }}/postgresql{{ postgresql_version_terse }}-check-db-dir {{ postgresql_data_directory }}
{% endif %}
```

Reply:
Indentation removed. version_compare() filter used to check the version.